CRM/Core/BAO/CustomField.php

```diff
@@ -1159,6 +1163,11 @@ public static function getDisplayValueCommon(
         $html_type == 'Multi-Select'
       )
     ) {
+      // Keep only the custom option 'key' => 'value' pair
+      // to avoid return of unnecessary data from the hook.
+      if (isset($option['attributes'])) {
+        unset($option['attributes']);
+      }
```

Comment:
Since `$option` is being passed by ref into this function, this may have side-effects.

Aside, the php `unset` function doesn't care if the variable exists or not, so there's no need to wrap it in a conditional checking `isset`.